### PR TITLE
Improvements about schema dumping [ci skip]

### PR DESCRIPTION
**Pull request by andreynering**

### CODE REVIEW

File: guides/source/active_record_migrations.md [hunk 1]
```diff
@@ -943,10 +949,10 @@ that Active Record supports. This could be very useful if you were to
 distribute an application that is able to run against multiple databases.
 
 There is however a trade-off: `db/schema.rb` cannot express database specific
-items such as triggers, or stored procedures. While in a migration you can
-execute custom SQL statements, the schema dumper cannot reconstitute those
-statements from the database. If you are using features like this, then you
-should set the schema format to `:sql`.
+items such as triggers, stored procedures or check constraints. While in a
+migration you can execute custom SQL statements, the schema dumper cannot
+reconstitute thosestatements from the database. If you are using features like
```
Comment (rafaelfranca) [inline guides/source/active_record_migrations.md]: missing space here

Comment (andreynering) [inline guides/source/active_record_migrations.md]: It's correct now :+1:

File: guides/source/active_record_migrations.md [hunk 2]
```diff
@@ -695,6 +696,11 @@ of `create_table` and `reversible`, replacing `create_table`
 by `drop_table`, and finally replacing `up` by `down` and vice-versa.
 This is all taken care of by `revert`.
 
+NOTE: If you want to add check constraints like in the examples above,
+you will have to use `structure.sql` as dump method. See
+[Schema Dumping and You](#schema-dumping-and-you).
+
+
```
Comment (seuros) [inline guides/source/active_record_migrations.md]: extra line here

Comment (andreynering) [inline guides/source/active_record_migrations.md]: Thank you!

**Comment by andreynering** [comment]
@rafaelfranca The Edge Guides wasn't updated since 11 days ago, just to let you know.

**Pull request merged**